book/text/solution/napajeni.tex

```diff
@@ -32,16 +32,16 @@ \section{Zvyšující měnič}
     \nonumber
 \end{eqnarray}
 
-Kde $t_{OFF}$ je čas po který je tranzistor sepnut, $t_{OFF}$ čas po který je tranzistor rozepnut, $T$ je perioda spínání a $s$ střída spínání.
+Kde $t_{OFF}$ je čas, po který je tranzistor sepnut, $t_{OFF}$ čas po který je tranzistor rozepnut, $T$ je perioda spínání a $s$ střída spínání.
 
-Ze vztahu je vidět, že napětí $U_2 \geq U_1$. Pokud použijeme reálné součástky, tak tato rovnost neplatí pokud je tranzistor trvale rozepnut, tak napětí $U_2$ bude menší o úbytek na diodě a cívce. Dále ze vztahu vyplývá, že velikost výstupního napětí je závislá jen na velikosti vstupního napětí a střídě spínání.
+Ze vztahu lze vyvodit, že platí $U_2 \geq U_1$. Při reálném použití tato rovnost neplatí, jestliže je tranzistor trvale rozepnut, je napětí $U_2$ menší o úbytek na diodě a cívce. Dále ze vztahu vyplývá, že velikost výstupního napětí je závislá pouze na velikosti vstupního napětí a střídě spínání.
```

Comment:
tady nechat reálné součástky, nebo možná neideální, ale ne reálné použití, protože ideální součástky se nedají použít reálně a kdyby dali, tak nejspíš by neměli parazitní vlastnosti, ale byli by ideální